pkg/sfu/forwarder.go

```diff
@@ -1585,7 +1601,7 @@ func (f *Forwarder) getTranslationParamsCommon(extPkt *buffer.ExtPacket, layer i
 	if f.lastSSRC != extPkt.Packet.SSRC {
 		if err := f.processSourceSwitch(extPkt, layer); err != nil {
 			tp.shouldDrop = true
-			return tp, err
+			return tp, nil
```

Comment:
Returning `nil` from here as `shouldDrop` is used. Also returning error flows through and `WriteRTP` logs an error which is not correct.